string command
float32 arg1
float32 arg2
float32 arg3
